.env

```diff
@@ -0,0 +1,7 @@
+source venv/bin/activate
```

Comment:
This file should not be pushed to github. It contains infomation that should be private